mergify_engine/actions/draft.py

```diff
@@ -0,0 +1,108 @@
+# -*- encoding: utf-8 -*-
+#
+#  Copyright © 2021 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import voluptuous
+
+from mergify_engine import actions
+from mergify_engine import check_api
+from mergify_engine import context
+from mergify_engine import rules
+from mergify_engine import subscription
+from mergify_engine import user_tokens
+from mergify_engine.actions import utils as action_utils
+from mergify_engine.rules import types
+
+
+class DraftAction(actions.Action):
+    flags = (
+        actions.ActionFlag.ALLOW_AS_ACTION
+        | actions.ActionFlag.ALWAYS_RUN
+        | actions.ActionFlag.ALLOW_ON_CONFIGURATION_CHANGED
+    )
+
+    validator = {
+        voluptuous.Required("bot_account", default=None): types.Jinja2WithNone,
+    }
+
+    async def run(
+        self, ctxt: context.Context, rule: rules.EvaluatedRule
+    ) -> check_api.Result:
+        if not ctxt.pull["draft"]:
```

Comment:
This should be clearer to remove on indent

```suggestion
        if ctxt.pull["draft"]:
            return Result(...)
```